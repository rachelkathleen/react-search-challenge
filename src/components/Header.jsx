import React from 'react';
import '../styles/Header.scss';

export default function Header() {
  return (
    <header className="header">
      <img src="./logo.svg" alt="match" />
    </header>
  );
}
