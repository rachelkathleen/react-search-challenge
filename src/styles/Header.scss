@import 'globals';

.header {
  border-bottom: 1px solid $light-gray;
  padding: 16px;

  img {
    width: 110px;
  }
}
