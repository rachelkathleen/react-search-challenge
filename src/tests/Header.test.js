import Header from '../components/header';
import React from 'react';
import { shallow } from 'enzyme';

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
