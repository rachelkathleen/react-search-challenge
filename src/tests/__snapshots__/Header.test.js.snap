// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <img
    alt="match"
    src="./logo.svg"
  />
</header>
`;
